BizzNEST/salinas-bizznest-scheduler
# [Feat]: Edit Pair

## Changes
1. Added +/- buttons to table rows
2. Incorporated the functionality of adding and removing interns to pairings 
3. Added a quick fix to the week card pairings table's header

https://github.com/user-attachments/assets/91fc1593-3043-4901-a200-7c8e6c736616



## Purpose
To be able to add and remove interns from pairings. (Manual Override)

## Approach
Intern Pairings are saved in a local storage so that they are accessible throughout the file. To add interns to a pairing the user can choose from a list of interns that have not been selected for a grouping. To remove an intern, the intern s taken from the pair grouping and added to the unselected group. Since this is a manual override pairings can be edited any way the user wants, no longer adhering to the filters from the generate function


## Testing Steps
- Add Interns, single/multiple at a time, testing various groups
- Remove interns, single/multiple at a time, testing various groups, and empty groups should be able to be deleted

## Learning

- [How to make a modal](https://www.w3schools.com/howto/howto_css_modals.asp)
- [Sets/Arrays](https://www.geeksforgeeks.org/how-to-convert-set-to-array-in-javascript/) 

Closes #72
